content/en/docs/releases/kubeflow-1.9.md

```diff
@@ -85,6 +85,16 @@ weight = 95
         </td>
       </tr>
       <!-- ======================= -->
+      <!-- Data Working Group -->
```

Comment:
My main concern is that WG Data charter has not been merged yet: https://github.com/kubeflow/community/pull/673